testcases/PowerNVDump.py

```diff
@@ -1068,6 +1068,72 @@ def runTest(self):
             else:
                 log.info("PASSED: Hugepage size is {} kB".format(hugepage_size))
 
+class KernelCrash_XIVE_off(PowerNVDump):
+    '''
+    This test checks kdump/fadump with kernel parameter option xive=off 
+    '''
+
+    def runTest(self):
+        self.cv_SYSTEM.goto_state(OpSystemState.OS)
+        self.setup_test()
+        log.info("=============== Testing kdump/fadump with xive=off ===============")
+        mmu = self.c.run_command("awk '$1 == \"MMU\" {print $3}' /proc/cpuinfo")[0]
+        log.debug(" MMU '{}'".format(mmu))
+        if mmu == "Radix":
```

Comment:
In case MMU isn't Radix, the test will silently pass without doing anything. Can we add else block and print a message like "Hash MMU detected, skipping the test"

Reply:
Done